lib/cluster/election.c

```diff
@@ -24,9 +24,9 @@
 
 struct pcmk__election {
     enum pcmk_ipc_server server;    // For message type
```

Comment:
The latest push re-added the `server` member

Reply:
doh